lib/krane/container_logs.rb

```diff
@@ -3,7 +3,7 @@ module Krane
   class ContainerLogs
     attr_reader :lines, :container_name
 
-    DEFAULT_LINE_LIMIT = 25
+    DEFAULT_LINE_LIMIT = 100
```

Comment:
I think most importantly this should be configurable, e.g. `DEFAULT_LINE_LIMIT = Integer(ENV.fetch('KRANE_LINE_LIMIT', 100))`.

Because one default won't work for all apps ever.